commands/InjectedCommandGroup.java

```diff
@@ -1,49 +1,38 @@
 package org.usfirst.frc4904.standard.commands;
 
+import edu.wpi.first.wpilibj2.command.CommandGroupBase;
 
-import edu.wpi.first.wpilibj.command.Command;
-import edu.wpi.first.wpilibj.command.CommandGroup;
+public abstract class InjectedCommandGroup extends CommandGroupBase {
+	private final CommandGroupBase previous;
 
-public abstract class InjectedCommandGroup extends CommandGroup {
-	private final Command previous;
-	
-	public InjectedCommandGroup(Command previous) {
-		super();
+	public InjectedCommandGroup(String name, CommandGroupBase previous) {
+		super(); // TODO: is this redundant? https://github.com/RoboticsTeam4904/standard/pull/254#discussion_r366097218
```

Comment:
no you need it for all commandbases and subystembases. Please remove comment in the code